var/spack/repos/builtin/packages/vigra/package.py

```diff
@@ -12,8 +12,10 @@ class Vigra(CMakePackage):
        customizable algorithms and data structures"""
 
     homepage = "https://ukoethe.github.io/vigra/"
+    git = "https://github.com/ukoethe/vigra.git"
     url = "https://github.com/ukoethe/vigra/releases/download/Version-1-11-1/vigra-1.11.1-src.tar.gz"
 
+    version('master', branch='master', preferred=True)
```

Comment:
Hmm, maybe can we avoid having a branch as preferred version?
```suggestion
    version('master', branch='master', preferred=True)
```